src/app/interfaces/event.ts

```diff
@@ -24,6 +24,17 @@ export interface CampusEventRequest extends CampusEvent {
 /** RESTful response interface for CampusEvent. */
 export interface CampusEventResponse extends CampusEvent {
   id: number;
+  create_time: string;
+  update_time: string;
+  num_enrolled: number;
+  deadline: string;
+  description: string;
+  name: string;
+  time: string;
+  location: string;
+  num_hours: number;
+  num_participants: number;
+  program: number;
```

Comment:
I think it's better to return program as an object as an extra field. For example, you can return the ID of the program as it is, and add another field on CampusEventSerializer, named `program_detail`, and this field serialize the program which the event is related to. In this way, we can display human-friendly information about the program in user's browser.

Reply:
program_detail should be a `Program` object, not `object`.